build_test.go

```diff
@@ -31,6 +29,16 @@ func testBuild(t *testing.T, context spec.G, it spec.S) {
 		buildpackYMLParser *fakes.BuildpackConfigParser
 
 		build packit.BuildFunc
+
+		layer = packit.Layer{
```

Comment:
This is a nitpick, but can you name this `helperLayer` just for test clarity, and then also define the variable at the top and set the value in the `it.Before` below so it matches the other variables?

Reply:
done